grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -65,7 +65,12 @@ class SelectOptionsApiController {
         List<PaymentTerm> paymentTerms = genericApiService.getList(PaymentTerm.class.simpleName, [sort: "name"]).collect {
             [id: it.id, label: it.name, value: it.id ]
         }
-        render([data: paymentTerms] as JSON)
+        Map blankPaymentTermOption = [
+                id: 'null',
+                value: 'null',
```

Comment:
I think it could be empty string

Reply:
unfortunately it is quite hard to implement this with an empty string since there are many places in our current filter logic where we check for a ''truthy" value of a filter.
Additionally I think having this option a `'null'` is more explicit and much more self explanatory.